### ISSUE TITLE
Backend - tasks to complete

### ISSUE THREAD

**Issue by sumanthvrao** [MicrosoftMobileInnovationLab/Mlab_website#2]
Some things to be implemented. *Sorted priority wise*
1) Displaying - projects by mentors / results (Maybe an **announcement** page for that)
2) Project selection by interns.
3) Progress Evaluation - Multiple rounds.
4) HashCode registration.
5) Integrate HashCode coupon-web application.
---
**Authentication**:
1) Manual creation for now (Mock USN)
2) OAuth using PES Credentials is a viable alternative.
---
Videos / posters / Slides - Probably on Object store (Microsoft / Firebase cloud)
Display using `iframe` for Youtube/ Slideshare.

**Comment by sumanthvrao** [comment]
Yeah! Yes i am asking people to take this as a hobby project. Yup :D

**Comment by aniketnk** [comment]
Let's look into [Firebase Cloud Functions](https://firebase.google.com/docs/functions/) so we don't have to worry about hosting the backend. Development would be much quicker too. \
We can run our own ExpressJS server on firebase using cloud functions. And it's FREE! 🤑

**Comment by sumanthvrao** [comment]
Awesome! Lets go with this. I'll at-least get to learn this from a [pro](https://github.com/aniketnk). :)
Will check it out after internals.

**Comment by aniketnk** [comment]
Cool, we can work on it together!
PS: I'm no **pro**! xP

**Comment by sumanthvrao** [comment]
Issue moved here https://github.com/MicrosoftMobileInnovationLab/Mlab_backend/issues/1
